zio-http/src/main/scala/zio/http/model/headers/values/Server.scala

```diff
@@ -0,0 +1,32 @@
+package zio.http.model.headers.values
+
+/**
+ * Server header value.
+ */
+sealed trait Server
+
+  object Server {
+    
+    /**
+     * A server value with a name
+     */
+    final case class ServerValue(name: String) extends Server
```

Comment:
Maybe call this `Name`?

Reply:
I changed it to ServerName, is your preference for "Name"?